packages/i18n/__demo__/index.html

```diff
@@ -12,9 +12,10 @@
     }
   </style>
   <body>
+    <!-- prettier-ignore -->
     <script type="module">
-      import 'https://cdn.jsdelivr.net/npm/benchmark@2.1.4/benchmark.min.js';
       import 'https://cdn.jsdelivr.net/npm/lodash@4.17.21/lodash.min.js';
+      import 'https://cdn.jsdelivr.net/npm/benchmark@2.1.4/benchmark.min.js';
```

Comment:
So does this means that to use benchmark.js as cdn, import order matters ? In that case, would it be better to just include it as a part of dev dependencies because we feature this in demo page anyways ?

Reply:
My opinion is if it's in demo page, it is being used every now and then. This is the only demo that import packages using cdn.

Reply:
Changed to dev dependencies. No noticeable build time impact. :)